steps/graph.py

```diff
@@ -1,27 +1,26 @@
 import json
-from typing import Dict, Optional
-
-from zquantum.core.graph import generate_barbell_graph as _generate_barbell_graph
-from zquantum.core.graph import generate_caveman_graph as _generate_caveman_graph
-from zquantum.core.graph import generate_graph_from_specs as _generate_graph_from_specs
-from zquantum.core.graph import generate_ladder_graph as _generate_ladder_graph
-from zquantum.core.graph import (
-    generate_random_graph_erdos_renyi as _generate_random_graph_erdos_renyi,
-)
-from zquantum.core.graph import (
-    generate_random_regular_graph as _generate_random_regular_graph,
-)
+from typing import Optional
+
+import zquantum.core.graph
 from zquantum.core.graph import save_graph
 
 
+def _make_sampler(random_weights: bool = False):
+    if random_weights:
+        return zquantum.core.graph.uniform_sampler()
+    else:
+        return zquantum.core.graph.static_sampler()
```

Comment:
I noticed that you elect to use this function and `random_weights` as opposed to using `create_object` along with `sampler_specs` in order to generate the samplers from the workflow steps. Is there a reason for this because it seems like it wouldn't be possible to generate certain samplers such as `choice_sampler` or `uniform_sampler` with different `min_val` and `max_val`.